File: Service/MediCare/RMA.Service.MediCare.Contracts/Entities/Medical/ModifierOutput.cs
using System;
using System.Collections.Generic;

namespace RMA.Service.MediCare.Contracts.Entities.Medical
{
    public class ModifierOutput
    {
        public string ModifierCode { get; set; }
        public string ModifierDescription { get; set; }
        public DateTime ModifierServiceDate { get; set; }
        public decimal ModifierQuantity { get; set; }
        public decimal ModifierAmount { get; set; }
        public decimal TotalIncusiveAmount { get; set; }
        public List<InvoiceLineDetails> ModifiedInvoiceLines { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/medicare/src/app/medical-invoice-manager/models/modifier-output.ts
import { InvoiceLineDetails } from 'projects/medicare/src/app/medical-invoice-manager/models/medical-invoice-line-details';

export class ModifierOutput{
    modifierCode: string;
    modifierDescription: string;
    modifierServiceDate: Date;
    modifierQuantity: number;
    modifierAmount: number;
    totalIncusiveAmount: number;
    modifiedInvoiceLines: InvoiceLineDetails[];
}